BusinessLayer/SongPlayer/SongPlayer.h

```diff
@@ -18,6 +18,7 @@
 #include <QScopedPointer>
 #include <QMediaMetaData>
 #include "../SongControl/SongControl.h"
+#include <QDebug>
```

Comment:
This include is already on line 6, we should also just not have QDebug included since we're not debugging in release.